client/gutenberg/extensions/tiled-gallery/layout/mosaic/ratios.js

```diff
@@ -27,12 +27,11 @@ export function ratioFromImage( { height, width } ) {
  * Build three columns, each of which should contain approximately 1/3 of the total ratio
  *
  * @param  {Array<number>} ratios Images ratios
+ * @param  {number} columnCount   Images ratios
```

Comment:
`Integer` instead of `number`?

Reply:
That's intentional, `Integer` isn't a JavaScript type. I did get the Array wrong though, jsdoc is funky…

Reply:
Fixed in 6bb7e2f